src/main-client.ts

```diff
@@ -890,6 +897,44 @@ export class MainClient extends BaseRestClient {
     );
   }
 
+  /**
+   *
+   * Staking Endpoints
+   *
+   **/
+
+  //TODO: https://binance-docs.github.io/apidocs/spot/en/#purchase-staking-product-user_data
+  //TODO: https://binance-docs.github.io/apidocs/spot/en/#redeem-staking-product-user_data
+  //TODO: https://binance-docs.github.io/apidocs/spot/en/#set-auto-staking-user_data
+
+  getStakingProductList(
```

Comment:
```suggestion
  getStakingProducts(
```

Sorry about the nitpicks, this is great & I really need to document what kind of standard I'm trying to set 😅